Formula/jid.rb

```diff
@@ -0,0 +1,22 @@
+class Jid < Formula
+  desc "jid"
```

Comment:
Thanks for your contribution! A more comprehensive description of what jid does would be welcome.